src/textual/_animator.py

```diff
@@ -141,7 +143,7 @@ def __call__(
 class Animator:
     """An object to manage updates to a given attribute over a period of time."""
 
-    def __init__(self, target: MessageTarget, frames_per_second: int = 60) -> None:
+    def __init__(self, target: App, frames_per_second: int = 60) -> None:
```

Comment:
Is the Animator only associated with the app?

If so, may as well rename `target` to `app` ?

Reply:
Done